Add RN for removal self link in managedObject parents fragment in res… (#2491)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10.20.589.0_Self-links-for-parent-assets-not-working.md

---

> date: 2024-10-08

Reviewer: ```suggestion
date: 
```

Author: Changed.

> title: Removed Self link from inventory parents fragment in API response

Reviewer: ```suggestion
title: Removed self link from parents fragments in Inventory API responses
```

Author: Changed.

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-JlFdtOPva
    label: Rest API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-60938
version: 10.20.589.0
---

> In the Inventory API, the `self` field has been removed from `deviceParents`, `assetParents` and `additionParents` response fragments. It was not required and always had a generated link to a non-existent endpoint.

Reviewer: ```suggestion
In order to simplify the Inventory API the redundant `self` field has been removed from `deviceParents`, `assetParents` and `additionParents` response fragments. It was not required and always had a generated link to a non-existent endpoint.
```

> The <code>self</code> field has been removed from inventory API from deviceParents, assetParents and additionParents response fragments, it was not required and always had a generated link to a non-existent endpoint.

Reviewer: ```suggestion
In the Inventory API, the `self` field has been removed from `deviceParents`, `assetParents` and `additionParents` response fragments. It was not required and always had a generated link to a non-existent endpoint.
```

Author: Changed.
